openmc/mgxs/library.py

```diff
@@ -792,6 +792,10 @@ def get_xsdata(self, domain, xsdata_name, nuclide='total', xs_type='macro',
             parameter is set to `True`.  In this case, this parameter sets the
             number of equally-spaced points in the domain of [-1,1] to be used
             in building the tabular distribution. Default is `33`.
+        subdomain : iterable of int
```

Comment:
Add a `NOTE` to the end of this docstring mentioning that `distribcell` subdomains are not yet supported. In the future, this should be relatively easy to implement using @smharper's distributed materials feature. In any case, the `subdomain` notation now corresponds to both `mesh` and `distribcell` spatial tally domains, so it would be good to indicate what is and is not supported.